ports/minizip-ng/portfile.cmake

```
@@ -8,18 +8,30 @@ vcpkg_from_github(
         Modify-header-file-path.patch
 )
 
+vcpkg_check_features(
+    OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+        zlib MZ_ZLIB
+        bzip2 MZ_BZIP2
+        lzma MZ_LZMA
+        zstd MZ_ZSTD
+        openssl MZ_OPENSSL
+)
+
 vcpkg_cmake_configure(
-    SOURCE_PATH ${SOURCE_PATH}
+    SOURCE_PATH "${SOURCE_PATH}"
     PREFER_NINJA 
     DISABLE_PARALLEL_CONFIGURE
     OPTIONS 
+        ${FEATURE_OPTIONS}
+        -DMZ_FETCH_LIBS=OFF
         -DMZ_PROJECT_SUFFIX:STRING=-ng
 )
 
 vcpkg_cmake_install()
 
 vcpkg_fixup_pkgconfig()
-vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/${PORT})
+vcpkg_cmake_config_fixup(CONFIG_PATH "lib/cmake/${PORT}")

```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/${PORT})
```
Since this is a relative path, it's unnecessary to add quotes.

---

I had stumbled upon the [style guide for internal scripts](https://github.com/microsoft/vcpkg/blob/master/docs/specifications/scripts-extraction.md#audit-points), which says “All non-splat variable expansions are in quotes ""”. I assume this simply does not apply to port scripts?

---

It does apply, and given the variable reference `${PORT}` I'm surprised by the request to remove them. `${PORT}` will never contain space or semicolon, but I think the rules mean to exclude simple words like `static` or `release`.
(It is *not* about relative paths.)

---

Keep them or delete them are both okay.